gravitee-apim-gateway/gravitee-apim-gateway-core/src/test/java/io/gravitee/gateway/jupiter/core/condition/ExpressionLanguageConditionFilterTest.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.jupiter.core.condition;
+
+import static io.gravitee.gateway.jupiter.core.condition.CompositeConditionFilterTest.MOCK_EXCEPTION;
+import static org.mockito.Mockito.when;
+
+import io.gravitee.definition.model.ConditionSupplier;
+import io.gravitee.el.TemplateEngine;
+import io.gravitee.gateway.jupiter.api.context.RequestExecutionContext;
+import io.reactivex.Maybe;
+import io.reactivex.observers.TestObserver;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@ExtendWith(MockitoExtension.class)
+class ExpressionLanguageConditionFilterTest {
+
+    protected static final String EXPRESSION = "test";
+
+    @Mock
+    private ConditionSupplier conditionSupplier;
+
+    @Mock
+    private RequestExecutionContext ctx;
+
+    @Mock
+    private TemplateEngine templateEngine;
+
+    final ExpressionLanguageConditionFilter<ConditionSupplier> cut = new ExpressionLanguageConditionFilter<>();
+
+    @Test
+    void shouldNotFilterWhenConditionEvaluatedToTrue() {
+        when(conditionSupplier.getCondition()).thenReturn(EXPRESSION);
```

Comment:
I would prefer to continue using the mock instead of creating an implementation. Don't see where it will be clearer than using the mock we are all used to use.

Reply:
Mocking a string is really overkilling. Using an actual implementation could bring you a better coverage and detach the code from the unit test.

Reply:
I've pushed changes